tests/Strava/API/Service/RESTTest.php

```diff
@@ -38,8 +41,8 @@ public function testGetAthlete()
                 ->with($this->equalTo('GET'), $this->equalTo('athlete'))
                 ->will($this->returnValue(new Response(200, [], '{"response": 1}')));
 
-            $service = new \Strava\API\Service\REST(static::TOKEN, $restMock);
-            $serviceWithVerbosity = new \Strava\API\Service\REST(static::TOKEN, $restMock, 1);
+            $service = new REST(static::TOKEN, $restMock);
```

Comment:
Import simplification is a side fix, main change is in method 'testUploadActivity'